### ISSUE TITLE
Unable to make

### ISSUE THREAD

**Issue by obscurecolin** [vellvm/vellvm#122]
Hi, I'm wanting to play around with this project but I encounter a build error each time I attempt to `make` it. 

I've installed the recommended coq version from the README (along with the listed dependencies):

> $ coqc -v
The Coq Proof Assistant, version 8.8.0 (January 2020)
compiled on Jan 29 2020 22:53:38 with OCaml 4.09.0

However, when I attempt to `make` it:

> make coq
make[1]: Entering directory '/home/colin/vellvm/src'
COQC coq/Denotation.v
COQC coq/Handlers/Memory.v
COQC coq/Handlers/UndefinedBehaviour.v
File "./coq/Handlers/UndefinedBehaviour.v", line 39, characters 0-15:
Error: PARAMS already exists.

> make[1]: *** [Makefile:70: coq/Handlers/UndefinedBehaviour.vo] Error 1
make[1]: Leaving directory '/home/colin/vellvm/src'
make: *** [Makefile:38: all] Error 2

As I understand it, a `PARAMS` Section is present in quite a few files and I've encountered no issues with previous definitions in the other files.

I apologise if this isn't really an "issue" with the project (as I could be to blame) or if I'm making an issue on a project that's still heavily under development (where the current master branch is a bit volatile), but any insight would be much appreciated as I'm fairly new to building Coq projects.

Thanks.

**Comment by YaZko** [comment]
Hello,
No worries, our build instructions leading to a failure more than qualifies as an Issue :)

This is on us sorry, we have not used 8.8 in a long time and our travis currently builds on Coq 8.9.1. I just noticed that allowing different sections with the same name in the same module is only allowed since 8.9.
I just pushed a quick patch that _should_ fix the problem. However I don't have an 8.8 installation under hand right now so if you do not mind bumping your version to 8.9, that would be simpler. Note that it also builds on 8.10 if you prefer (but not on 8.11 that got released today as far as I know). 

I will test the dependencies more properly asap and update accordingly the instructions.

Thanks for the report and sorry for the hiccup!
